tensorflow/python/kernel_tests/array_ops_test.py

```diff
@@ -648,6 +648,18 @@ def testEllipsis(self):
       with self.assertRaisesRegexp(ValueError, "Multiple ellipses"):
         _ = checker[..., :, ...].eval()
 
+  @test_util.run_deprecated_v1
+  def testFetchMulti(self):
+    with self.session(use_gpu=True):
+      raw = [[[1,2,3], [4, 5, 6]], [[7,8,9], [10, 11, 12]]]
+      checker = StridedSliceChecker(self, raw)
+
+      _ = checker[[0, 1, 1], [1, 0, 1], [1, 0, 2]]
```

Comment:
Can some of these be tensors?

Reply:
Yes.

Reply:
yes

Reply:
@alextp, From my latest commit, It could support any index within the slice_spec could be `list` or `np.ndarray` or `ops.Tensor`. Here is the example:
```
>>> import tensorflow as tf
>>> import numpy as np
>>> tf.enable_eager_execution()
>>> foo = tf.constant([[1,2,3], [4,5,6], [7,8,9]])
>>> index_a = tf.constant([0, 1])
>>> index_b = tf.constant([1, 0])
>>> index_arr_a = np.array([0, 1])
>>> index_arr_b = np.array([1, 0])
>>> print(foo[[0, 1], [1, 0]])
tf.Tensor([2 4], shape=(2,), dtype=int32)
>>> print(foo[[0, 1], index_b])
tf.Tensor([2 4], shape=(2,), dtype=int32)
>>> print(foo[index_a, index_b])
tf.Tensor([2 4], shape=(2,), dtype=int32)
>>> print(foo[index_a, index_arr_b])
tf.Tensor([2 4], shape=(2,), dtype=int32)
>>> print(foo[[0, 1], index_arr_b])
tf.Tensor([2 4], shape=(2,), dtype=int32)
>>> print(foo[index_arr_a, index_arr_b])
tf.Tensor([2 4], shape=(2,), dtype=int32)
```
And already pass all the test in my local Mac.

Reply:
What I want are tests that verify that calling convert_to_tensor on the indices and passing them through has the same meaning.

Reply:
I see, I will update it right now.